src/xenia/kernel/user_module.cc

```diff
@@ -101,7 +102,8 @@ X_STATUS UserModule::LoadFromMemory(const void* addr, const size_t length) {
     }
 
     // Copy the xex2 header into guest memory.
-    const xex2_header* header = this->xex_module()->xex_header();
+    auto header = this->xex_module()->xex_header();
+    auto security_header = this->xex_module()->xex_security_info();
```

Comment:
no need for this here - can just use the local xex_module variable (here and below)